src/main/flight/imu.c

```diff
@@ -503,7 +507,7 @@ static void imuDebug_GPS_RESCUE_HEADING(void)
         if (magYaw < 0) {
             magYaw += 3600;
         }
-        DEBUG_SET(DEBUG_GPS_RESCUE_HEADING, 4, magYaw); // mag heading in degrees * 10
+//        DEBUG_SET(DEBUG_GPS_RESCUE_HEADING, 4, magYaw); // mag heading in degrees * 10
```

Comment:
Why is this commented out?

Reply:
Thank you.  I was using debug 4 for other purposes and forgot to restore it.
Debugs are still 'fluid' at this point - many were configured for development rather than end-user tuning etc.  Need to resolve what we actually need to retain once the code is finalised, vs what we need to get the development done.